tests/test_plugin.py

```diff
@@ -260,3 +260,13 @@ def test_get_available_testruns(api_client, tr_plugin):
     testplan_id = 100
     api_client.send_get.return_value = TESTPLAN
     assert tr_plugin.get_available_testruns(testplan_id) == [59, 61]
+
+
+def test_close_test_run(api_client, tr_plugin):
+    api_client.send_get.return_value = {'is_completed': False}
+    assert tr_plugin.is_testrun_available() is True
+    
+    tr_plugin.close_test_run()
```

Comment:
Sorry but here you are not testing `close_test_run` but `is_testrun_available` :)
We should instead test the POST query triggered by `close_test_run`.
Look examples at [line 174](https://github.com/Igur007/pytest-testrail/blob/94bdac8cb8ae2c7199464f6bfe39089886a7d6be/tests/test_plugin.py#L174) or [line 202](https://github.com/Igur007/pytest-testrail/blob/94bdac8cb8ae2c7199464f6bfe39089886a7d6be/tests/test_plugin.py#L202)